src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -55,8 +55,7 @@ const SignInPageContent = props => (
             <LoginKeyboardAvoidingView
                 behavior="position"
                 contentContainerStyle={[
-                    props.isSmallScreenWidth ? styles.signInPageNarrowContentMargin : {},
-                    !props.isMediumScreenWidth ? styles.signInPageWideLeftContentMargin : {},
+                    StyleUtils.getLoginContainerMarginStyles(props),
```

Comment:
@metehanozyurt
On second thoughts, Let's revert the last two commits, just stick to the existing one itself. 
```js
                    props.isSmallScreenWidth ? styles.signInPageNarrowContentMargin : {},
                    !props.isMediumScreenWidth || (props.isMediumScreenWidth && props.windowHeight < variables.minHeightToShowGraphics) ? styles.signInPageWideLeftContentMargin : {},
```

Reply:
I'm doing it right now. Thanks.